### PR TITLE
Add fastaddjob

### PR DESCRIPTION
**Pull request by hwamir**

### CODE REVIEW

File: disq/client.py [hunk 1]
```diff
@@ -388,6 +388,13 @@ def addjob(self, queue, body, timeout_ms=0, replicate=0, delay_secs=0,
             args += [Token('ASYNC')]
 
         return self.execute_command(*args)
+        
+    def fastaddjob(self, queue, body, timeout_ms=0):
+        """
+        addjob without optional arguments.
+        """
+        args = ['ADDJOB', queue, body, timeout_ms]
+        return self.execute_command(*args)
```
Comment (ryansb) [inline disq/client.py]: You can actually write this as `return self.execute_command('ADDJOB', queue, body, timeout_ms)` and skip allocating the `args` list.

**Comment by ryansb** [comment]
Thanks for the PR, a simple addjob is definitely needed. 

I'm thinking about dividing into two clients, both using DisqueAlpha under the hood. One would be SimpleDisque, a more pythonic client that would have convenience methods like fastaddjob, and a RawDisque that would have methods directly mapping to the disque commands (currently DisqueAlpha)

File: disq/client.py (hunk 1 above)
Comment (hwamir) [inline disq/client.py]: You are definitely right. I`ll fix it.

**Comment by hwamir** [comment]
Provide pythonic client is really good idea.

I`ll close this PR until you make another client.
Thank you.

**Pull request closed without merging**